### PR TITLE
added new leading type "embedded" to component <Leading />. Built action and reducer for event.

### PR DESCRIPTION
**Pull request by chiangkeith**
- added new leading type "embedded" to component <Leading />
- Built action and reducer for event.

### CODE REVIEW

File: src/components/Leading.js [hunk 1]
```diff
@@ -28,6 +28,20 @@ export default class Leading extends Component {
 
   render() {
     const { leading, mediaSource } = this.props
+    const embed = mediaSource.embed
+    const eventPeriod = mediaSource.eventPeriod? mediaSource.eventPeriod : [ null, null ]
```
Comment (hcchien) [inline src/components/Leading.js]: could you have a space before the question mark? :)

File: src/components/Leading.js [hunk 2]
```diff
@@ -28,6 +28,20 @@ export default class Leading extends Component {
...
+    const flag = mediaSource.flag? mediaSource.flag : 'NA'
```
Comment (hcchien) [inline src/components/Leading.js]: 'NA'? or null?

File: src/actions/articles.js [hunk 3]
```diff
@@ -319,6 +344,11 @@ function _buildTopicQueryUrl(params = {}, slug= '') {
   return formatUrl(`topics/${slug}?${query}`)
 }
 
+function _buildEventQueryUrl(params = {}, slug= '') {
+  let query = _buildQuery(params)
+  return formatUrl(`event/${slug}?${query}`)
+}
+
 function _buildImageQueryUrl(params = {}) {
```
Comment (imkojima) [inline src/actions/articles.js:352]: Do we still need `slug` here?

File: src/components/Leading.js (hunk 1 above)
Comment (chiangkeith) [reply src/components/Leading.js]: yes, already have them look better

File: src/components/Leading.js (hunk 2 above)
Comment (chiangkeith) [reply src/components/Leading.js]: null sounds better

File: src/actions/articles.js (hunk 3 above)
Comment (chiangkeith) [reply src/actions/articles.js:352]: maybe no, already remove that

**Pull request merged**